modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/index.js

```diff
@@ -69,6 +69,20 @@ export function isValuesArrayChanged(prevValue = [], newValue = []) {
 	return changed;
 }
 
+export function getLocalizableLabel(label, fallback) {
+	if (!label) {
+		return fallback ?? '';
+	}
+
+	return (
+		label[Liferay.ThemeDisplay.getLanguageId()] ??
+		label[Liferay.ThemeDisplay.getDefaultLanguageId()] ??
+		fallback ??
+		label['en_US'] ??
```

Comment:
Fallback should be the last one before the empty string so that we exhaust all the possibilitites before falling back